packages/core/admin/admin/src/pages/MarketplacePage/components/NpmPackageCard/PackageStats.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Box } from '@strapi/design-system/Box';
+import { Typography } from '@strapi/design-system/Typography';
+import { Flex } from '@strapi/design-system/Flex';
+import { Icon } from '@strapi/design-system/Icon';
+import Github from '@strapi/icons/Github';
+import Download from '@strapi/icons/Download';
+import Star from '@strapi/icons/Star';
+import { Divider } from '@strapi/design-system/Divider';
+
+const VerticalDivider = styled(Divider)`
+  width: 12px;
+  transform: rotate(90deg);
+`;
+
+const PackageStats = ({ githubStars, weeklyDownloads }) => {
+  return (
+    <Flex>
+      {!!githubStars && (
```

Comment:
I'm not sure if it is intentional: `0` would fail this test.

Reply:
yes, it is as. We don't have to show 0 stars.